packages/compiler/src/core/checker.ts

```diff
@@ -2932,8 +2932,20 @@ export function createChecker(program: Program, resolver: NameResolver): Checker
         if (base.flags & SymbolFlags.Alias) {
           base = getAliasedSymbol(base, undefined);
         }
-        if (base) {
+        if (base && (base.exports || base.members)) {
```

Comment:
why do you need to check exports and members?

Reply:
and i think you should check Sep first, if it's :: then goto the metaProperty part